### DISCUSSION TITLE
Is there a way to insert raw request data without having to import cURL to see the raw option

### DISCUSSION THREAD

**Discussion by ozsoft** [hoppscotch/hoppscotch#1758] [Q&A] [answered]
Is there a way to insert raw request data without having to import cURL to see the raw option. I couldnt find it anywhere but I can see the option when I include 'data' section with a cURL import.

**Comment by liyasthomas** [comment, marked as answer]
1, Select any method type that accepts `raw request body`.
2. Turn on "Raw input" toggle.
3. Add your raw request.

Screenshot:

<img width="1440" alt="Screenshot 2021-08-04 at 4 23 02 PM" src="https://user-images.githubusercontent.com/10395817/128169505-74c218cf-f011-4230-94fc-da6fafdf8709.png">
